src/Propel/Runtime/ActiveRecord/NestedSetRecursiveIterator.php

```diff
@@ -80,15 +95,19 @@ public function next()
             }
             $this->curNode = $nextNode;
         }
-
-        return $this->curNode;
```

Comment:
I'm a little bit confused by this one. Why we removed this?

Reply:
Check php core method docblock 
It says it cannot return and any return is ignored.